activity2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<table>
+    <rows>
+        <header>Empno</header>
+        <header>Empname</header>
+        <header>Salary</header>
+    </rows>
```

Comment:
should have used these for element names instead of making separate row
```
<row>
    <empno></empno>
    <empname></empname>
    <salary></salary>
</row>
```